sdk/python/feast/infra/registry/remote.py

```diff
@@ -48,13 +49,12 @@ def __init__(
         repo_path: Optional[Path],
         auth_config: AuthConfig = NoAuthConfig(),
     ):
-        auth_header_interceptor = GrpcClientAuthHeaderInterceptor(auth_config)
         self.auth_config = auth_config
         channel = grpc.insecure_channel(registry_config.path)
-        self.intercepted_channel = grpc.intercept_channel(
-            channel, auth_header_interceptor
-        )
-        self.stub = RegistryServer_pb2_grpc.RegistryServerStub(self.intercepted_channel)
+        if self.auth_config.type is not AuthType.NONE.value:
```

Comment:
why `is not` instaed of `!=`?

Reply:
No real reason. I can update. Just moved the code from old file to here.